en/lessons/basics/collections.md

```diff
@@ -3,120 +3,224 @@ version: 1.3.1
 title: Collections
 ---
 
-Lists, tuples, keyword lists, and maps.
+Tuples, Lists, Assoc Lists, Sets, Maps and Hashmaps.
```

Comment:
It might be useful to mention `Vector` and `Seq` as well?

Reply:
I'm not sure about Seq. From what I hear and see (and use), it's more or less at the same level as  `Array i e ` nowadays…

Reply:
I ended up not covering seq. I don't personally use it very often, and don't understand it very well. If someone wants to add it in a subsequent PR I think that is fine.